Readme.md

```diff
@@ -2,6 +2,12 @@
 
 **Fast, unopinionated, minimalist web framework for [Node.js](http://nodejs.org).**
 
+<<<<<<< docs/add-scorecard-badge
```

Comment:
reminder about the conflcit markers

Reply:
Solved in https://github.com/expressjs/express/pull/5436/commits/4d8aa0692bd21c1ee292d66ee7a708da36d7442f